web/src/features/moreCast2/slices/yesterdayDailiesSlice.ts

```diff
@@ -22,8 +22,7 @@ const yesterdayDailiesSlice = createSlice({
   reducers: {
     getYesterdayDailiesStart(state: State) {
       state.error = null
-      state.yesterdayDailies = []
-      state.loading = true
+      state.loading = false
```

Comment:
why did the value of `loading` get changed to `false` here?

Reply:
It's a bug putting up a fix with tests now :)